### PR TITLE
feat: Changes about encrypted backups

### PR DESCRIPTION
**Pull request by sharat87**
## Description 

Adds details about encrypted backups that include the salt/password in it.

Preview page at https://appsmith-docs-git-feat-encrypted-backups-get-appsmith.vercel.app/getting-started/setup/instance-management/appsmithctl.

## Type of PR

Check the appropriate box:

- [ ] Review Fixes
- [ ] Documentation Overhaul
- [x] Feature/Story
    - Link one or more Engineering Tickets
        * https://github.com/appsmithorg/appsmith/pull/29902
- [ ] A-Force
- [ ] Error in documentation

## Documentation tickets
 Link to one or more documentation tickets:
 
 - 

## Checklist
Choose only the ones that are applicable.

- [ ] Checked for Grammarly suggestions.
- [ ] Adhered to the writing checklist.
- [ ] Adhered the media checklist.
- [ ] Verified and updated cross-references or added redirect rules.
- [ ] Tested the redirect rules on deploy preview.
- [ ] Validated the modifications made to the content on deploy preview.
- [ ] Validated the CSS modifications on different screen sizes.

### CODE REVIEW

**Comment by jnikhila** [comment]
@sharat87 - I've rewritten the content. Please take a look and let me know if any changes are needed.

File: website/docs/getting-started/setup/instance-management/appsmithctl.mdx [hunk 1]
```diff
@@ -39,64 +39,77 @@ Subcommands allow you to trigger different operations like exporting or importin
...
+
+      The above command creates a backup without encryption, and the backup archive won't include `APPSMITH_ENCRYPTION_PASSWORD` and `APPSMITH_ENCRYPTION_SALT` environment variables in the `docker.env` file. Make sure to note them separately if you choose this option.
+
+    The backup archive file is stored in the `/appsmith-stacks/data/backup/` folder in your container directory. You can access it at `./stacks/data/backup/` on your local machine.
+
+3. If you can't access or have a different volume configuration, copy the archive file to your host disk with:
+
+   ```bash
+   # Replace the appsmith-backup-DATE_AND_TIMESTAMP.tar.gz with the backup file name
+   docker cp appsmith:/appsmith-stacks/data/backup/appsmith-backup-DATE_AND_TIMESTAMP.tar.gz .
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-management/appsmithctl.mdx:70]: Encrypted backups end with `.tar.gz.enc`. Should we mention that as well?

File: website/docs/getting-started/setup/instance-management/appsmithctl.mdx [hunk 2]
```diff
@@ -39,64 +39,77 @@ Subcommands allow you to trigger different operations like exporting or importin
...
+      #highlight-next-line
+      kubectl exec ANY_APPSMITH_POD_NAME appsmithctl backup
+      ```
 
-```bash
-#highlight-next-line
-kubectl cp ANY_APPSMITH_POD_NAME:/appsmith-stacks/data/backup/APPSMITH_BACKUP_FILENAME -n APPSMITH_NAMESPACE GIVE_NAME_FOR_COPIED_BACKUP_FILE
-```
-After running the above command, the backup archive is copied to your current directory, indicating that the backup process is completed. You can now proceed with the remaining steps of the process you were performing.
+      The above command encrypts the backup archive with a password. When prompted, enter the password and ensure you remember it. You need this password if you ever want to restore from this backup. If you forget it, you won't be able to restore this backup.
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-management/appsmithctl.mdx:93]:
```suggestion
      The above command encrypts the backup archive with a password. When prompted, enter a password and ensure you remember it. You need this password if you ever want to restore from this backup. If you forget it, you won't be able to restore this backup.
```

File: website/docs/getting-started/setup/instance-management/appsmithctl.mdx [hunk 3]
```diff
@@ -39,64 +39,77 @@ Subcommands allow you to trigger different operations like exporting or importin
...
-```bash
-#highlight-next-line
-kubectl exec ANY_APPSMITH_POD_NAME appsmithctl backup
-```
-The archive file is stored in the container directory `/appsmith-stacks/data/backup/`.
+    * **With encryption** - To create a backup with encryption use the below command (replace the `ANY_APPSMITH_POD_NAME` with your Appsmith pod name):
 
-4. Copy the archive file using the below command, ensure that you replace the `ANY_APPSMITH_POD_NAME` with your Appsmith pod name, APPSMITH_BACKUP_FILENAME with your Appsmith backup filename, and APPSMITH_NAMESPACE with your Appsmith namespace:
+      ```bash
+      #highlight-next-line
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-management/appsmithctl.mdx:89]: There's only one line in this code block, no need to highlight it.

```suggestion
      # Replace ANY_APPSMITH_POD_NAME with the name of a running Appsmith pod.
```

File: website/docs/getting-started/setup/instance-management/appsmithctl.mdx [hunk 4]
```diff
@@ -39,64 +39,77 @@ Subcommands allow you to trigger different operations like exporting or importin
...
-#highlight-next-line
-kubectl cp ANY_APPSMITH_POD_NAME:/appsmith-stacks/data/backup/APPSMITH_BACKUP_FILENAME -n APPSMITH_NAMESPACE GIVE_NAME_FOR_COPIED_BACKUP_FILE
-```
-After running the above command, the backup archive is copied to your current directory, indicating that the backup process is completed. You can now proceed with the remaining steps of the process you were performing.
+      The above command encrypts the backup archive with a password. When prompted, enter the password and ensure you remember it. You need this password if you ever want to restore from this backup. If you forget it, you won't be able to restore this backup.
+
+    * **Without encryption** - To create a backup without encryption use the below command (replace the `ANY_APPSMITH_POD_NAME` with your Appsmith pod name):
+
+      ```bash
+      #highlight-next-line
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-management/appsmithctl.mdx:98]: There's only one line in this code block, no need to highlight it.

```suggestion
      # Replace ANY_APPSMITH_POD_NAME with the name of a running Appsmith pod.
```

File: website/docs/getting-started/setup/instance-management/appsmithctl.mdx [hunk 5]
```diff
@@ -39,64 +39,77 @@ Subcommands allow you to trigger different operations like exporting or importin
...
+      ```
+
+      The above command creates a backup without encryption, and the backup archive won't include `APPSMITH_ENCRYPTION_PASSWORD` and `APPSMITH_ENCRYPTION_SALT` environment variables in the `docker.env` file. Make sure to note them separately if you choose this option.
+
+    The archive file is stored in the container directory `/appsmith-stacks/data/backup/`.
+
+3. Copy the archive file using the below command, ensure that you replace the `ANY_APPSMITH_POD_NAME` with your Appsmith pod name, APPSMITH_BACKUP_FILENAME with your Appsmith backup filename, and APPSMITH_NAMESPACE with your Appsmith namespace:
+
+   ```bash
+   #highlight-next-line
```
Comment (sharat87) [inline website/docs/getting-started/setup/instance-management/appsmithctl.mdx:109]: There's only one line in this code block, no need to highlight it.

```suggestion
      # Replace placeholders in capitals with appropriate values based on your deployment.
```

File: website/docs/getting-started/setup/instance-management/appsmithctl.mdx (hunk 1 above)
Comment (jnikhila) [reply website/docs/getting-started/setup/instance-management/appsmithctl.mdx:70]: Is the above `appsmith-backup-DATE_AND_TIMESTAMP.tar.gz` not correct?

**Review by jnikhila — APPROVED** [review]

**Pull request merged**